index.html

```diff
@@ -1364,24 +1388,47 @@ <h3 id="encryption">Encryption</h3>
           >.
         </p>
 
+        <h3 id="multiple_measurements">Multiple measurements of the same type</h3>
+        <p>
+          If you want to send multiple measurements of the same type, e.g. three temperatures,
+          you can just add multiple measurements of the same type to the payload. A
+          <code>device_id</code> (1, 2, 3 etc.) will be added in the order of which you define
```

Comment:
This is the first time we mention device_id. That is a Python parser specific implementation and the spec itself shouldn't care.

Reply:
Yes, you're right. Removed the device_id and only mention now that a postfix will be added to the measurement